staff/views.py

```diff
@@ -267,11 +289,147 @@ def make_announcement(request):
     return render(request, "make_announcement.html", {"form": form})
 
 
+@group_required("staff")
+def post_form(request):
+    if request.method == "POST":
+        form = PostFormForm(request.POST, request.FILES)
+        if form.is_valid():
+            posted_form = Form(
+                name=form.cleaned_data.get("name"),
+                description=form.cleaned_data.get("description"),
+                form=request.FILES["form"],
+                created_by=DjangoUser.objects.get(id=request.user.id),
+            )
+            if form.cleaned_data.get("esign") is not None:
+                posted_form.esign = form.cleaned_data.get("esign")
+            posted_form.save()
+            posted_form.recipient_groups.set(form.cleaned_data.get("recipient_groups"))
+            posted_form.recipient_classrooms.set(
+                form.cleaned_data.get("recipient_classrooms")
+            )
+            messages.add_message(request, messages.SUCCESS, "Successfully Posted Form")
+            data = request.POST.copy()
```

Comment:
Why does this get a `copy()`?

Reply:
Fixed in ffd66b3af3e9a66fe7b8673b0066ffb410ce6268